### PR TITLE
add myself to manifest/sample/test owner

### PR DESCRIPTION
**Pull request by rmgogogo**
PR is moved from https://github.com/kubeflow/pipelines/pull/2893/files
That PR has CLA issue while after I follow the guide, it still can't be fixed. So copy out a new PR and submit

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3027)

### CODE REVIEW

File: manifests/kustomize/OWNERS [hunk 1]
```diff
@@ -5,3 +5,4 @@ approvers:
 reviewers:
   - Bobgy
   - IronPan
+  - rmgogogo
```
Comment (Bobgy) [inline manifests/kustomize/OWNERS]: Just a confirmation, will you have capacity to review all of these?

Comment (rmgogogo) [reply manifests/kustomize/OWNERS]: looks strange you ask this

Comment (Bobgy) [reply manifests/kustomize/OWNERS]: I don't get it. I meant you are adding yourself as reviewer for a lot of folders, that seems a lot of work to review all those PRs

Comment (rmgogogo) [reply manifests/kustomize/OWNERS]: The reason is detailed in https://github.com/kubeflow/pipelines/pull/2893/files.
As tech leader and hosted owner, I need to well track the possible breaking changes, especially preloaded samples and manifest. Test (Rui's work can help a lot) also should be well take care. Our post-submit always fail now. Manifest is not well managed. We don't have sense to make the manifest change be back-compatible. Any of these would make hosted broken. Rui's test can help but test can't cover all things. Put me to reviewer list to keep an eye is a good approch. As for workload, definitelly still lever dedicated owners for a detailed review.

Comment (Bobgy) [reply manifests/kustomize/OWNERS]: @rmgogogo Thanks for the explanation! Sounds good to me.

**Pull request merged**